classlib/bytecoder.web/src/main/java/de/mirkosertic/bytecoder/api/web/HTMLCanvasElement.java

```diff
@@ -15,7 +15,9 @@
  */
 package de.mirkosertic.bytecoder.api.web;
 
-public interface HTMLCanvasElement extends HTMLElement {
+public interface HTMLCanvasElement extends HTMLElement, CanvasImageSource {
 
     CanvasRenderingContext2D getContext(final String contextName);
+    int width();
```

Comment:
I guess the width() and height() method should be annotated with '@OpaqueProperty', because they are Properties on JS-Side, right? Else a method invocation would be generated, which might not be good.

Reply:
If the name doesn't differ, it wasn't necessary, right?
Anyway it's more neat and I'm of course fine with it!

Reply:
Bytecoder needs to know that this is a property access, and not a method invocation. This is stated by the annotation, so it is required to generate correct code.

Reply:
Okay, clear, I will change it in a minute